spec/models/transaction_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Transaction, type: :model do
+  # before {
+  #   customer = Customer.create!(first_name: "Logan", last_name: "Wyatt")
```

Comment:
lets use factories instead of creating objects this way. It'll be way faster as we build more and more test cases.

Reply:
Cool, you'll have to show us more about that, we made these to ensure that our 16 length validation was working

Reply:
For sure, we can group call later and discuss how to implement FactoryBot + Faker.